ctags/parsers/asciidoc.c

```diff
@@ -54,11 +55,13 @@ static NestingLevels *nestingLevels = NULL;
 static NestingLevel *getNestingLevel(const int kind)
 {
 	NestingLevel *nl;
+	tagEntryInfo *e;
 
 	while (1)
 	{
 		nl = nestingLevelsGetCurrent(nestingLevels);
-		if (nl && nl->type >= kind)
+		e = getEntryOfNestingLevel (nl);
+		if ((nl && (e == NULL)) || (e && (e->kind - AsciidocKinds) >= kind))
```

Comment:
Because it can be set to CORK_NIL in makeAsciidocTag() when vStringLength (name) == 0.